styleguide/backstop.json

```diff
@@ -423,12 +420,6 @@
       "referenceUrl": "https://americanmedicalassociation.github.io/ama-style-guide-2/patterns/02-molecules-promo-promo-with-image/02-molecules-promo-promo-with-image.html",
       "selectors": ".ama__promo--background"
     },
-    {
-      "label": "promo as homepage",
-      "url": "http://localhost:3000/patterns/02-molecules-promo-promo-as-homepage/02-molecules-promo-promo-as-homepage.html",
-      "referenceUrl": "https://americanmedicalassociation.github.io/ama-style-guide-2/patterns/02-molecules-promo-promo-as-homepage/02-molecules-promo-promo-as-homepage.html",
-      "selectors": ".ama__promo--homepage"
```

Comment:
Can you comment on why this one is removed as well?

Reply:
@damontgomery I didn't realize I removed that test.